extensions/amp-analytics/analytics-vars.md

```diff
@@ -361,3 +371,75 @@ Provides the number of seconds that have elapsed since 1970. (Epoch time)
 
 Example value: `1452710304312`
 
+## Visibility Variables
+
+### backgrounded
```

Comment:
Are the below two variables related to prerendering or separate. It might be good to note either way. For example, when could a page be backgroundedAtStart?